test/testcafe/framework/page-objects/SelectAuthenticatorPageObject.js

```diff
@@ -46,28 +61,38 @@ export default class SelectFactorPageObject extends BasePageObject {
     return elCount === 1;
   }
 
+  getFactorCTAButtonByIndex(index) {
+    return this.form.getElement(factorSelectButtonSelector).nth(index);
+  }
+
   getFactorSelectButtonByIndex(index) {
-    return this.form.getElement(factorSelectButtonSelector).nth(index).textContent;
+    return this.getFactorCTAButtonByIndex(index).textContent;
   }
 
   getFactorSelectButtonDataSeByIndex(index) {
     return this.form.getElement(factorSelectButtonDiv).nth(index).getAttribute('data-se');
   }
 
   async selectFactorByIndex(index) {
-    await this.t.click(this.form.getElement(factorSelectButtonSelector).nth(index));
+    await this.t.click(this.getFactorCTAButtonByIndex(index));
   }
 
   async skipOptionalEnrollment() {
```

Comment:
How difficult would it be to rename this to be more reflective of the action it takes?
```suggestion
  async clickSetUpLaterButton() { // or something like this
```